e2e/tests/truffle/test/Event.js

```diff
@@ -14,8 +14,9 @@ const {
 const EventTestContract = artifacts.require('EventTestContract')
 
 contract('EventTestContract', async (accounts) => {
-    let contract, from, nodeAddr, web3js, contractAddress
-
+    let contract, from, nodeAddr, contractAddress, web3eth
+    const newValueSetEventTopic = "0xb922f092a64f1a076de6f21e4d7c6400b6e55791cc935e7bb8e7e90f7652f15b"
+    const anotherValueSetEventTopic = "0xccdec039614c54f7047ec6e5fbd3b6b8d5d78e1a38737dcc140cf162a774a83f"
```

Comment:
Can you pull these from the ABI instead of hardcoding?

Reply:
Done